readme.md

```diff
@@ -47,6 +47,7 @@ This is a collection of **awesome resources** about [Prisma](https://www.prisma.
 - [@prisma-tools/admin](https://prisma-tools.ahmedelywa.com/admin/generator)
 - [Prisma plugin for the IntelliJ Platform](https://github.com/gabrielcolson/intellij-prisma)
 - [Gatsby with prisma-client](https://github.com/LekoArts/gatsby-with-prisma-client)
+- [ctrlplusb/prisma-pg-jest](https://github.com/ctrlplusb/prisma-pg-jest)
```

Comment:
I'd use the text for something more descriptive, e.g. "Testing Prisma with PostgreSQL and Jest"

Reply:
Done 😃